agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3798,12 +3800,44 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             break;
         }
 
-        // CMDU Message
-        auto notification = message_com::create_vs_message<
-            beerocks_message::cACTION_CONTROL_SLAVE_JOINED_NOTIFICATION>(cmdu_tx);
+        auto apconfHeader = cmdu_tx.create(0, ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WIFI_SIMPLE_CONFIGURATION_MESSAGE);
+        auto tlvAp = cmdu_tx.addClass<wfa_map::tlvApRadioBasicCapabilities>();
+        if (tlvAp == nullptr) {
+            LOG(ERROR) << "Error creating TLV_AP_RADIO_BASIC_CAPABILITIES";
+        }
+        for (int i=0; i<6; i++) {
+            tlvAp->radio_uid().mac[i] = network_utils::mac_from_string(config.radio_identifier).oct[i]; // TODO: put mac_from_string in variable
```

Comment:
Instead of this loop you should use:
`std::copy_n(network_utils::mac_from_string(config.radio_identifier).oct, MAC_ADDR_LEN,  tlvAp->radio_uid().mac)`

BTW: the standard doesn't define that the RUID==mac. Yes. it's the same size as MAC but not exclusive to that.
Please review the YAML file of this TLV to make sure it is up with the standard (**section 17.2.7  AP Radio Basic Capabilities TLV Format**)